hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1675,6 +1675,11 @@ func reconcileHostedControlPlane(hcp *hyperv1.HostedControlPlane, hcluster *hype
 		hyperv1.EtcdPriorityClass,
 		hyperv1.EnsureExistsPullSecretReconciliation,
 		hyperv1.TopologyAnnotation,
+		hyperv1.CertifiedOperatorsCatalogImageAnnotation,
```

Comment:
sooo... the old annotations didn't work at all since we weren't copying them to the HCP?

Reply:
yes, unfortunately in #2947 I verified them directly acting on the HCP instead of the HC. My fault.